src/drf_yasg/utils.py

```diff
@@ -227,8 +227,7 @@ def is_list_view(path, method, view):
     """
     # for ViewSets, it could be the default 'list' action, or an @action(detail=False)
     action = getattr(view, 'action', '')
-    method = getattr(view, action, None) or method
```

Comment:
When looking at git blame the `or method` was added to silence the linter. I don't think `method` should be stomped on like this since it should be a string (also confirmed in the docstring), and `str.detail` will always be none.

Reply:
Debugging this it looks like `view` has a `detail` for the test project's Article view set decorated with `@action`, and I think this was probably a typo, and instead of preserving `getattr(getattr(view, action, None), 'detail', None)` which won't work for the common case, I'm suggesting fixing this to `getattr(view, 'detail', None)` which for sure has a value.